src/Business Foundation/App/NoSeries/src/Single/NoSeriesImpl.Codeunit.al

```diff
@@ -429,6 +429,13 @@ codeunit 304 "No. Series - Impl."
         exit(OriginalNoSeriesCode);
     end;
 
+    procedure IsNoSeriesInDateOrder(NoSeriesCode: Code[20]): Boolean
+    var
+        NoSeries: Record "No. Series";
+    begin
+        exit(NoSeries.Get(NoSeriesCode) and NoSeries."Date Order")
```

Comment:
I'm not sure this is universal.

Shouldn't it throw an error if not found?

Reply:
implemented in a similar way to:
procedure IsAutomatic(NoSeriesCode: Code[20]): Boolean
    var
        NoSeries: Record "No. Series";
    begin
        if not NoSeries.Get(NoSeriesCode) then
            exit(false);
        exit(NoSeries."Default Nos.");
    end;